bin/OLI01/conf.mx6qsabrelite_na

```diff
@@ -0,0 +1,24 @@
+# Permission is hereby granted, free of charge, to any person obtaining a
```

Comment:
Interesting. I hadn't actually expected people to contribute their board configurations; I figured this was more of an example that people could use than a shared project.

However, It's nice to see people using this:-) and I see no reason not to take these config file patches. I'll wait until the comments on the previous/other patch are addressed though, since this will need slight updates too.

BTW, you should add a copyright line to the top of these files, otherwise the text "the above copyright notice" in the license text doesn't make sense.